portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -76,7 +76,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
               (long) entity.getKey().getId(),
               (long) entity.getProperty("likeCount"),
               (String) entity.getProperty("email"),
-              (float) entity.getProperty("sentimentScore"))
+              (float)(double) entity.getProperty("sentimentScore"))
```

Comment:
might make sense to store sentimentScore as a double in Comment?

Reply:
ack :) I need to manually round a double on the front end, while a float just gives you a nice one decimal number by default.